src/RoleObjects/MathData.js

```diff
@@ -0,0 +1,19 @@
+import SectionData from './SectionData.js';
+import React from 'react';
+
+/**
+ * Base class for role objects that use the math HTML tag.  This contains only setters/getters for fields that are common to all math tags regardless of the type attribute.
```

Comment:
Nit: this isn't a base class since it is not intended to have subclasses, and there isn't a math tag in HTML